backend/test_observer/controllers/test_executions/test_executions.py

```diff
@@ -99,6 +104,26 @@ def start_test_execution(
         raise HTTPException(status_code=400, detail=str(exc)) from exc
 
 
+@router.put("/end-test")
+def end_test_execution(request: EndTestExecutionRequest, db: Session = Depends(get_db)):
+    test_execution = (
+        db.query(TestExecution)
+        .filter(TestExecution.ci_link == request.ci_link)
+        .one_or_none()
+    )
+
+    if test_execution is None:
+        raise HTTPException(status_code=404, detail="Related TestExecution not found")
+
+    failed = any(r.status == C3TestResultStatus.FAIL for r in request.test_results)
+
+    if failed:
+        test_execution.status = TestExecutionStatus.FAILED
+    else:
+        test_execution.status = TestExecutionStatus.PASSED
+    db.commit()
```

Comment:
sorry if it's a dumb question, but is it okay that we don't return any response here?

Reply:
By default if we don't return anything at least the response would be 200 if it works